packs/wix.js

```diff
@@ -0,0 +1,25 @@
+  /** Additional description of a Lighthouse audit that tells the user to be aware of offscreen images, and low quality image placeholder behavior, in the context of the Wix CMS platform. */
+  "offscreen-images": "Images are automatically lazy loaded in most scenarios. Offscreen images may be downloaded below the viewport. These are very small, low-resolution placeholder images which are fast to load. As the user scrolls down the page, these placeholders are replaced by high-resolution images. This serves the same purpose as lazy loading. [Learn more](https://support.wix.com/en/article/site-performance-how-wix-improves-your-sites-loading-time#lazy-loading-images) about how Wix improves your site's loading time.",
+  /** Additional description of a Lighthouse audit that tells the user to defer loading of non-critical third-party libraries, in the context of the Wix CMS platform. */
+  "render-blocking-resources": "Wix ensures that our resources are not render-blocking. When [using third-party code](https://support.wix.com/en/article/site-performance-using-third-party-code-on-your-site) on your site, it’s important to add it in a non-blocking manner. For example, you can add defer or async attributes to your third-party script. You can also place the script at the end of the code body to ensure that your regular site content doesn’t get delayed or prevented from rendering.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by avoiding animated images and using videos where possible, in the context of the Wix CMS platform. */
+  "efficient-animated-content": "Using GIFs can significantly slow down a page's loading speed. Where possible, use videos instead of GIFs to showcase animated content. Modern video encoding formats are both higher quality and smaller in size than GIF animations. [Learn more](https://support.wix.com/en/article/site-performance-optimizing-your-media#optimizing-videos-on-your-site) about optimizing your site's media.",
+  /** Additional description of a Lighthouse audit that tells the user to be aware of JavaScript code that is not used, particularly from third-parties, in the context of the Wix CMS platform. */
+  "unused-javascript": "Third-party code is one of the main reasons for unused JavaScript. Identify these files and review whether they're necessary to your site. If not, consider removing or replacing them with an optimized alternative. Any code delivered through static.parastorage.com is packaged by Wix. It's crucial to your site's structure, so it can't be modified or deleted. We are consistently optimizing these files using methods such as code splitting, dynamic imports and using web workers to reduce main thread blocking JavaScript in order to minimize input delays and make sure that sites load quickly, regardless of their complexity. [Learn more](https://support.wix.com/en/article/site-performance-optimizing-your-media#optimizing-videos-on-your-site).",
```

Comment:
What tools does wix provide for users that are failing this audit? Telling users to remove unused services is pretty general advice that Lighthouse already suggests. The part about code splitting/dynamic imports doesn't identify any action the user can take.

Also I think the link is incorrect here.